test/TestsHelper.js

```diff
@@ -393,6 +393,7 @@ TestsHelper.testTooManyArgs = function testTooManyArgs(baseCmd, additionalArgsCo
 TestsHelper.execCmd = function execCmd(cliCmd, options, done) {
   options = options || {
     env: {
+      PATH: process.env.PATH,
```

Comment:
What's the purpose of this?

Reply:
The node variable was not found in Travis - https://travis-ci.org/Kinvey/kinvey-cli/builds/352367292.

Reply:
Alright. 
Just one more thing - you have to add the `PATH` to the `options`. Currently, if the `options` argument is null or undefined, the `PATH` won't be added and you'd have to add it in multiple places.